bot.js

```diff
@@ -1,12 +1,21 @@
-// IIFE (Immediatly Invoked Function Expression) to set up variables
+const dotenv = require('dotenv');
+dotenv.config();
+
+
+// IIFE (Immediately Invoked Function Expression) to set up variables
 (function() {
 	global.Discord = require('discord.js')
 	global.fs = require('fs')
-	global.client = new Discord.Client()
+	// bot intents
+	myintents = new Discord.Intents(Discord.Intents.NON_PRIVILEGED)
+	myintents.add('GUILD_MEMBERS')
+	global.client = new Discord.Client({ws: {intents: myintents}})
```

Comment:
intents dont need to be declared in the script

Reply:
They need to be declared on client instantiation if I wish to use them; I use them to display nicknames in memberstzs.js. Do you know where else I could put them?

Reply:
It should be fine there then.